spec/services/open_library_service_spec.rb

```diff
@@ -0,0 +1,28 @@
+require 'rails_helper'
+
+RSpec.describe 'GET to Openlibrary service' do
```

Comment:
you need to define, what is the class that you makes a test
Like
```ruby
describe User, type: :model

#or

describe UserController, type: :controller
```
in this case you need to describe a Service